draft-rescorla-tls-esni.md

```diff
@@ -109,7 +109,15 @@ when, and only when, they appear in all capitals, as shown here.
 # Overview
 
 This document is designed to operate in one of two primary topologies
-shown below, which we call "Shared Mode" and "Split Mode"
+shown below, which we call "Shared Mode" and "Split Mode".
+
+In both cases, DNS records point to the provider's client-facing server.
+A client-facing server may operate in either mode simultaneously, depending
+on which domain is being served.
+
+In principle, the provider might not be the origin for any domains. As far as
+the client is concerned, it is the logical origin for a large set of
+innocuous domains, while also providing protection for some private domains.
```

Comment:
Use of words like "origin" and "retrieves content" might sound like that the use of HTTP is implied. It might make sense avoid using such words, and rather talk about things like "server that terminates TLS" or "application data".

Reply:
This is the text for the common case between shared and split mode. The origin server has different behavior in each case. 

In shared mode the client-facing-server terminates TLS; the origin may be elsewhere. In split mode, the origin behind the client-facing-server is the server that terminates TLS. That distinction is made below, not here.

This text was moved, with minor modifications, from it's original location. The term "origin" is frequently used within the CDN community, and has a well-understood meaning. I purposely used retrieves content, because the origin server may very well only use HTTP, if it's a private server protected by this client-facing server.  But I can change it to application data; I'm not sure it really maters.